closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -41,6 +41,8 @@ else ifeq ($(call isTargetOs, linux), true)
     OPENJCEPLUS_JGSKIT_PLATFORM := s390-linux64
   else ifeq ($(call isTargetCpu, x86_64), true)
     OPENJCEPLUS_JGSKIT_PLATFORM := x86-linux64
+  else ifeq ($(call isTargetCpu, aarch64), true)
+    OPENJCEPLUS_JGSKIT_PLATFORM := arm-linux64
```

Comment:
Please continue to consider targets in alphabetical order.

Reply:
updated